server/handlers/requestHandlers.go

```diff
@@ -74,34 +74,49 @@ func (h *Handlers) LoginRequest(w http.ResponseWriter, r *http.Request) {
 	}
 
 	log.Println("INFO [handlers/requestHandlers.go] Log In Successful")
+	h.createToken(w, r, login.Email)
+	http.Redirect(w, r, "/api/home", http.StatusPermanentRedirect)
```

Comment:
I'm not sure why this doesn't work

Reply:
why do we have StatusPermanentRedirect/StatusTemporaryRedirect?

Reply:
Also, I change method from GET to POST in the /api/home handler, seems to work. Can you try it and see if it makes a difference

Reply:
i checked and it works. thanks kate